monitor/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "start": "node index.js",
     "lint": "eslint . --ignore-path ../.eslintignore",
-    "lint:fix": "eslint . --fix"
+    "lint:fix": "eslint . --fix",
+    "import:contracts": "rm -r -f ./abis && cp -r ../contracts/build/contracts ./abis/"
```

Comment:
Does it make sense to copy the ABIs in three different sub-directories locating in the same directory? What if we change paths to ABIs in the sources as so they will work with `../contracts/build/contracts` directly instead of local copy of this directory?

Reply:
I agree - done.

Reply:
This required me to disable `node/no-unpublished-require` lint rule and override `ModuleScopePlugin` in `ui`